site/app/controllers/student/SubmissionController.php

```diff
@@ -447,11 +447,18 @@ private function ajaxUploadSplitItem() {
             return $this->uploadResult("Invalid CSRF token.", false);
         }
 
+        // check for whether the item should be merged with previous submission,
+        // and if so, whether or not file clobbering should be done.
         $merge_previous = false;
+        $clobber = false;
         if(isset($_REQUEST['merge'])){
-            if($_REQUEST['merge']  === "true"){
+            if($_REQUEST['merge'] === "true") {
                 $merge_previous = true;
             }
+
+            if(isset($_REQUEST['clobber']) && $_REQUEST['clobber'] === "true") {
+                $clobber = true;
+            }
         }
```

Comment:
Condense this all to:
```
$merge_previous = isset($_REQUEST['merge']) && $_REQUEST['merge'] === 'true';
$clobber = isset($_REQUEST['clobber']) && $_REQUEST['clobber'] === 'true';
```
We require `$merge_previous` to be true to use `$clobber`, so we don't need to care about checking the isset of merge for setting.